docdown/__init__.py

```diff
@@ -2,4 +2,4 @@
 
 __author__ = """Jason Emerick"""
 __email__ = 'jason@mobelux.com'
-__version__ = '0.2.6'
+__version__ = '__version__ = '__version__ = '__version__ = '0.2.7''''
```

Comment:
```suggestion
__version__ = '0.2.7'
```

Reply:
Jason, see my last two commits.

Reply:
👍 once the CI passes, I will get this merged for you